coreservices/Dockerfile

```diff
@@ -3,36 +3,24 @@ FROM tensorchord/pgvecto-rs:pg16-latest
 
 # Define arguments that can be passed at build time
 ARG POSTGRES_USER
-ARG POSTGRES_DB_NAME
+ARG POSTGRES_DB
 ARG POSTGRES_PASSWORD
 ARG POSTGRES_PORT
 ARG POSTGRES_BASE_VERSION
 
 # Set environment variables from the arguments
 ENV POSTGRES_USER=${POSTGRES_USER:-postgres} \
-    POSTGRES_DB_NAME=${POSTGRES_DB_NAME:-postgres} \
+    POSTGRES_DB=${POSTGRES_DB:-postgres} \
     POSTGRES_PORT=${POSTGRES_PORT:-5432} \
     POSTGRES_BASE_VERSION=${POSTGRES_BASE_VERSION:-16.1} \
     POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
 
-# Install dos2unix with no cache and clean up
-RUN apt-get update -qq && \
-    apt-get install -y dos2unix && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-# Copy init.sql script to the container and convert line endings
-COPY init.sql /docker-entrypoint-initdb.d/init.sql
-RUN dos2unix /docker-entrypoint-initdb.d/init.sql
-
-# Copy init-db.sh script to the container and convert line endings
-COPY init-db.sh /docker-entrypoint-initdb.d/init-db.sh
-RUN dos2unix /docker-entrypoint-initdb.d/init-db.sh
-
-# Make the init-db.sh script executable
-# This is necessary because the script needs execute permission to run inside the container.
-# The chmod command changes the file mode to make it executable (x) for the user (owner of the file).
-RUN chmod +x /docker-entrypoint-initdb.d/init-db.sh
+# Copy the configuration file into the image
+COPY ./files/postgres/postgresql.conf /etc/postgresql/postgresql.conf
 
+# Expose the port (this does not publish the port; it's for documentation purposes)
 # Expose the port the app runs on
 EXPOSE ${POSTGRES_PORT}
+
+# Set the command to run PostgreSQL with the custom configuration file
+CMD ["postgres", "-c", "config_file=/etc/postgresql/postgresql.conf"]
```

Comment:
**Suggestion:** Consider adding a health check to the Dockerfile to ensure the PostgreSQL service is running correctly after the container starts.
> 🌟  **High Priority** 
 > [category: **`best practice`**, importance: **`8`**]
> **Reason:** Adding a health check to the Dockerfile is a best practice that ensures the PostgreSQL service is running correctly after the container starts. This is a significant improvement for container health monitoring. 

```suggestion
HEALTHCHECK --interval=30s --timeout=10s --start-period=5s --retries=3 CMD pg_isready -U $POSTGRES_USER -d $POSTGRES_DB || exit 1
CMD ["postgres", "-c", "config_file=/etc/postgresql/postgresql.conf"]
```